website/content/commands/services/deregister.mdx

```diff
@@ -36,6 +36,8 @@ This flexibility makes it easy to pair the command with the
 
 @include 'http_api_namespace_options.mdx'
 
+@include 'http_api_partitions_options.mdx'
```

Comment:
There's a mix of spellings here, some of these refs say `http_api_partitions_options` while others say `http_api_partition_options`. The filename with the partials is singular.